sdk/communication/communication-phone-numbers/swagger/README-operatorConnect.md

```diff
@@ -0,0 +1,15 @@
+# Azure Communication Services OperatorConnect Module
```

Comment:
Out of curiosity: what's the pros of splitting the swagger files into `-operatorConnect.md`, `-sipRouting.md` and so forth? I'm wondering if this "packaged" SDKs within the same SDK is the intended goal.

Reply:
Mainly autorest md files split was done so that we can granularly generate needed parst of sdk. As for the approach overall i'm open for discussion, we can reiterate on it once more in wider audience.

Reply:
From a SDK developer point of view having only one unified swagger is the simplest to maintain. But I guess that's hard because the services are separate?
I think you can reference multiple swagger files with one md file, the autorest generation time shouldn't be too long and you only ever regenerate when there is a new service api version.

Reply:
With the current setup in the PR running `rushx generate:client` will not generate all code and one has to manually alter the package.json or add a new command. The commands in package.json are quite standardized across the entire monorepo, so I'd recommend to have the command generate all clients at once. This only takes a few seconds and isn't worth optimizing.

Reply:
Updated "generate:client" command. Few notes:
- i kept separate md files for now due to few reasons: it seems that spec md files currently contain partial config for autorest with conditional logic based on tag (this is quite questionable approach as it seems more reasonable to keep all settings in one place)
- i had to add clean-up command before generation as "clear-output-folder: false" is set (otherwise executions break each other);
- I have not changed output of phone numbers related generated files "/generated/phoneNumbers", as was agreed, for now. I believe it has to be done as separate PR;
- As part of separate PR we can resolve all issues above by moving all settings into single md file in sdk repo and updating phonenumbers client generation settings ;